CanHazFunny/CanHazFunny.Tests/JokeServiceTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using NUnit.Framework;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+
+namespace CanHazFunny.Tests;
+public class JokeServiceTests
+{
+    [Test]
+    public void GtJoke_ReturnJoke()
```

Comment:
```suggestion
    public void GetJoke_ReturnJoke_NotNull()
```
Just a small typo with GetJoke, I would also add what outcome you are wanting to the end of the name of the test method

Reply:
Will fix

Reply:
Fixed